parlai/agents/bart/modules.py

```diff
@@ -25,3 +26,27 @@ def output(self, tensor: torch.Tensor) -> torch.Tensor:
         # project back to vocabulary
         output = F.linear(tensor, self.embeddings.weight)
         return output
+
+    def decode_forced(
+        self, encoder_states: Tuple[Any, ...], ys: torch.LongTensor
+    ) -> Tuple[torch.FloatTensor, torch.LongTensor]:
+        """
+        Decode with a fixed, true sequence, computing loss.
+
+        Override to bypass assertion that BOS is not present, and additionally insert
```

Comment:
nit: maybe "Overriding TGA.decode_forced() to bypass"? At first I thought this was a command, i.e. "You, the user, should override this method if you want to bypass..."